src/main/webapp/js/instructor.js

```diff
@@ -149,11 +149,42 @@ function setupFsCopyModal() {
             success: function(data) {
                 $('#courseList').html(data);
                 $('#fscopy_submit').prop('disabled', false);
+                $('#fsCopyModal form').submit(validateFsCopyModalSubmission);
             }
         });
     });
 }
 
+/**
+ * Prevents submission of the fsCopyModal if the same fsname is used in the original course, and the original course is 
+ * a destination of copying
+ */
+function validateFsCopyModalSubmission(e) {
+    var $fsCopyModal = $("#fsCopyModal");
+
+    var $checkedCourses = $fsCopyModal.find("[name='copiedcoursesid']:checked");
+    var checkedCoursesId =  $checkedCourses.map(function() {
+        return $(this).val();
+    });
+    var originalCourseId = $fsCopyModal.find("[name='courseid']").val();
+    var isOriginalCourseSelected = $.inArray(originalCourseId, checkedCoursesId) !== -1;
+
+    var fsName = $fsCopyModal.find('#copiedfsname').val();
+    var originalFsName = $fsCopyModal.find("input[name='fsname']").val();
+    var isSameFsName = fsName === originalFsName;
+
+
+    if (isOriginalCourseSelected && isSameFsName) {
```

Comment:
I think it would be best to not even let them select the course (disable the checkbox and show a message?) if a session with the same name already exists in it. 
Might be better to do just one issue as the rest would be wasted work?